docusaurus/docs/ruby/troubleshooting.mdx

```diff
@@ -442,69 +442,75 @@ You have a couple of options:
 
 ## Rake tasks under tests are run more than once in Queue Mode
 
-Make sure the Rake task is loaded once for each test example in the spec file:
+**`knapsack_pro` version 8.1.1 fixed this issue.**
```

Comment:
recommedation 

We could be more actionable. We could tell users what to do instead of them trying to figure it out.

```suggestion
Please update the `knapsack_pro` gem to the latest version.
```